e2e/app/gaspump.go

```diff
@@ -30,8 +31,10 @@ func DeployGasApp(ctx context.Context, def Definition) error {
 		return errors.Wrap(err, "deploy gas station")
 	}
 
-	if err := fundGasStation(ctx, def); err != nil {
-		return errors.Wrap(err, "fund gas station")
+	if def.Testnet.Network != netconf.Mainnet {
+		if err := fundGasStation(ctx, def); err != nil {
+			return errors.Wrap(err, "fund gas station")
+		}
```

Comment:
Maybe `if def.Testnet.Nework.IsEphemeral()`

Reply:
That will disable it for Omega too.